.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 
-go: 1.13
+go: "1.13"
```

Comment:
why ? it works both ways. 
https://docs.travis-ci.com/user/languages/go/#specifying-a-go-version-to-use

Reply:
Had a weak warning in goland.